### ISSUE TITLE
Implement support for readonly property on input fields

### ISSUE THREAD

**Issue by beeman** [formly-js/ngx-formly#732]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request
```

**Current behavior**

There is currently no way to mark an input field as being `readonly`, which is described [here](https://www.w3schools.com/tags/att_input_readonly.asp).

**Expected behavior**

The expected behavior is that a user of ngx-formly can define wether an input is set to `readonly` by passing in `readonly: true` in the `templateOptions` object.

**Comment by aitboudad** [comment]
fixed and released in `2.0.0-rc.28`.

**Comment by beeman** [comment]
Dude, amazing! So fast!! Will test it tomorrow at work!
